backend/src/services/contact/contact.service.ts

```diff
@@ -91,10 +86,12 @@ export class ContactService {
       ...payload.contact,
       organization: payload.organization,
     });
-
+    // eslint-disable-next-line
+    const { organization, statuses, ...rest } = result;
```

Comment:
Just change this part to this for preventing the error:
```js
// using `_` and `__` to prevent eslint error and doing the object destructuring safely
    const { organization: _, statuses: __, ...rest } = result;
```